projects/demo/src/modules/components/badged-content/index.html

```diff
@@ -0,0 +1,53 @@
+<tui-doc-page
+    header="BadgedContent"
+    package="KIT"
+    type="components"
+>
+    <ng-template pageTab>
+        <p>BadgedContent is a wrapper for other components to add badges and notifications to them.</p>
+
+        <tui-doc-example
+            *ngFor="let example of examples; let index = index"
+            [component]="index + 1 | tuiComponent"
+            [content]="index + 1 | tuiExample"
+            [description]="index === 1 && description"
```

Comment:
it can become cumbersome when there are several descriptions

can we leave it like this or think about how we can make it more beautiful?

Reply:
this is ok for one description, but if several maybe some interface to use in cycle `examples`

`
interface TuiExample {
    name: string;
    description: string | PolymorpheusContent;
}
`

Reply:
yes, let's leave it like this 

for more complex cases we can simply write examples without the loop